src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -1,8 +1,48 @@
 package traffic_simulation
 
+// Imports for parsing library
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import java.io.*
+
 fun main(args: Array<String>) {
 
-    scenario(testList, testRoad)
+    // The information of vehicles and their interest to drive is given in a csv-file
```

Comment:
Es ist normal, dass Eingaben zu parsen, viel Code braucht. Deswegen ist es eine gute Idee, die den Code der das macht in eine eigene Methode zu stecken. Rein geht der Dateiname, raus kommt eine Liste mit Fahrzeugen.